### ISSUE TITLE
Rotate MD80 still not working . 

### ISSUE THREAD

**Issue by midi48** [skiselkov/BetterPushbackC#162]
Hi just tried Push back with latest version and MD80 still spins around.

https://www.youtube.com/watch?v=qNB2yRkUivU&feature=youtu.be

[Log.txt](https://github.com/skiselkov/BetterPushbackC/files/1426592/Log.txt)

**Comment by slgoldberg** [comment]
Yes, still happens for me, too. Happy to test if you have an update you think solves this, and/or help with more information if needed. Thanks as always.

**Comment by slgoldberg** [comment]
D'oh, I spoke too soon. I should've tested with the latest version before posting -- and I just did tested with v0.46 using XP11.10r1 with BetterPushbackC v0.46 with Rotate-MD-80-XP11.acf v1.31 and it seems to be working.

Admittedly, I only tested it once on v0.46 -- and it did strangely stop pushing back midway through.  But when it paused (no failures that I heard or saw), I simply toggled the brakes on/off and it seemed to continue where it left off and finished the push-back (albeit pretty slowly).

In other words, though it was a little slow and seemed to pause mid-pushback, the plane didn't do "donuts" and ended up in the position and direction I had requested.

So I'm not sure if you intended to fix it with a recent change, or if I just got lucky this time, but anyway there's a data point if it's helpful.

Thanks again.

**Comment by midi48** [comment]
Seems to be working fine now, after last update. Thanks

**Comment by slgoldberg** [comment]
Yes, I tested it again (twice) last night, and it worked perfectly both times -- no spinning, no delay (as mentioned above), all good. So it seems fine on the latest 11.10r1 release of XP11 and your latest release. Glad you have a copy though so you can see for yourself next time. :-) . Thanks again.  -Steve
